eva-remapping-ingest/src/main/java/uk/ac/ebi/eva/remapping/ingest/batch/tasklets/RemappingMetadata.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.remapping.ingest.batch.tasklets;
+
+import org.springframework.data.annotation.Id;
+import uk.ac.ebi.ampt2d.commons.accession.hashing.SHA1HashingFunction;
+
+public class RemappingMetadata {
+
+    @Id
+    private String hashedMessage;
+
+    private String remappingVersion;
+
+    private String accessioningVersion;
```

Comment:
For documentation sake, suggest adding comments what we mean by remapping and accessioning versions.